Earthfile

```diff
@@ -79,11 +79,18 @@ smoke-test:
   FROM +smoke-init
   WORKDIR /docusaurus2
   RUN yarn global add fs-extra jest
-  COPY ./tests/e2e/specs ./__tests__/e2e/specs
+  COPY ./tests/e2e/specs/cli.spec.js ./__tests__/e2e/specs/cli.spec.js
   COPY ./tests/helpers ./__tests__/helpers
   COPY ./tests/e2e/jest.config.js ./jest.config.js
   ENV NODE_ENV=ci
   RUN node --expose-gc /usr/local/bin/jest --logHeapUsage --runInBand
+  COPY ./tests/e2e/docusaurus2-graphql-doc-generator-multi-instance.config.js ./docusaurus2-graphql-doc-generator-multi-instance.config.js
+  COPY ./tests/e2e/docusaurus2-graphql-doc-generator-multi-instance-1.config.js ./docusaurus2-graphql-doc-generator-multi-instance-1.config.js
+  COPY ./tests/e2e/docusaurus2-graphql-doc-generator-multi-instance-2.config.js ./docusaurus2-graphql-doc-generator-multi-instance-2.config.js
+  COPY ./scripts/config-plugin-multi-instance.js ./config-plugin-multi-instance.js
+  COPY ./tests/e2e/specs/cli-multi-instance.spec.js ./__tests__/e2e/specs/cli.spec.js
+  RUN node config-plugin-multi-instance.js
+  RUN node --expose-gc /usr/local/bin/jest --logHeapUsage --runInBand
```

Comment:
I do not understand why both `cli-multi-instance.spec.js` and `cli.spec.js` cannot be run at once.

Reply:
2 reasons:
1. `config-plugin.js` generates a `docusaurus.config.js` file, and `config-plugin-multi-instance.js` generates a different `docusaurus.config.js`. Single instance tests should run with the former and multi instance with the latter.
2. The code checks if the schema has changed to decide whether to generate the docs. This might make the tests flaky if they run in parallel because they take as input the same schema files. For example, if the multi-instance finishes first then the "should return 0 with generated message when completed as first run" test might fail (since this is no longer the first run).

Reply:
Since the multi-instance is now aligned with Docusaurus, ie one command per instance. I suggest merging both tests. The aim being to check that a config without `id` will run correctly, and a second config with`id` will not conflict with the default config. I hope this make sense.

Reply:
Good point. Addressed in 0d26176.